### PR TITLE
Feature/add sylius 1.12 support

### PR DESCRIPTION
**Pull request by mpysiak**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | yes
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: src/Controller/Helper/FormErrorsFlashHelper.php [hunk 1]
```diff
@@ -12,20 +12,21 @@
...
+use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
 final class FormErrorsFlashHelper implements FormErrorsFlashHelperInterface
 {
-    /** @var FlashBagInterface */
-    private $flashBag;
+    private $requestStack;
+
+    /** @var RequestStack */
```
Comment (senghe) [inline src/Controller/Helper/FormErrorsFlashHelper.php:22]:
```suggestion
    /** @var RequestStack */
    private $requestStack;
```

**Review by senghe — APPROVED** [review]

**Comment by Jos-Dalhuisen** [comment]
When will this be merged? We would like to update to Sylius 1.12.

**Comment by elboletaire** [comment]
Yayy! Thanks!

**Comment by sdrenth** [comment]
@mpysiak @senghe When will the new version be released which includes these changes?

**Pull request merged**